terraform-modules/aws/loki-stack/main.tf

```diff
@@ -0,0 +1,89 @@
+locals {
+  name = "loki-stack"
+}
+
+resource "aws_s3_bucket" "loki-stack" {
+  bucket = "${local.name}-${var.cluster_name}"
+  acl    = "private"
+
+  versioning {
+    enabled = true
+  }
```

Comment:
Can you add this in to enable encryption by default?  Basically every place i go to...they want this:  https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket#enable-default-server-side-encryption

Reply:
Added